include/rmlab/auxiliary/UUID.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef UUID_H
+#define UUID_H
+
+#pragma once
+
+#include <sstream>
+#include <random>
+#include <string>
+#include <iostream>
+using namespace std;
+
+namespace rmlab
+{
+namespace auxiliary
+{
+
+
+unsigned int random_char() {
```

Comment:
can you please add doxygen strings and move the implementation details to a `.cpp` file?